src/common/import_session.c

```diff
@@ -230,18 +230,20 @@ const char *dt_import_session_name(struct dt_import_session_t *self)
   return self->vp->jobcode;
 }
 
-
+/* This returns a unique filename using session path **and** the filename.
+   If current is true we will use the original filename otherwise use the pattern.
+*/
 const char *dt_import_session_filename(struct dt_import_session_t *self, gboolean current)
```

Comment:
current -> use_filename

(I understand this code was there before, but better use a more meaningful name while we are at it).

Reply:
done